pytorch_toolkit/object_detection/face_detection.md

```diff
@@ -6,7 +6,9 @@ Models that are able to detect faces on given images.
 | --------------------------- | ------------------- | --------- | -------------------- |---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
 | face-detection-retail-0005  | 0.982               | 1.021     | 84.52%               | 78.97% / 70.80% / 37.32%    | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/wider_face_tiny_ssd_075x_epoch_70.pth), [configuration file](./configs/face-detection-retail-0005.py) | 2       |
 | face-detection-0100         | 0.785               | 1.828     | 86.16%               | 82.50% / 75.77% / 40.83%    | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/face-detection-0100.pth), [configuration file](./configs/face-detection-0100.py)                      | 2       |
+| face-detection-0101         | 0.52                | 2.31      |                      | 85.13% / 76.63% / 42.87%    | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/face-detection-0101.pth), [configuration_file](./configs/face-detection-0101.py)                      | 4       |
 | face-detection-0102         | 1.767               | 1.842     | 91.47%               | 88.90% / 83.51% / 49.58%    | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/face-detection-0102.pth), [configuration file](./configs/face-detection-0102.py)                      | 2       |
+| face-detection-0103         | 1.71                | 4.46      |                      | 89.47% / 83.19% / 50.38%    | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/face-detection-0103.pth), [configuration_file](./configs/face-detection-0103.py)                      | 4       |
```

Comment:
@dqawami I'm sorry that it is not described in documentation, but for other models Complexity (GLFOPs) is measured by one of OpenVINO tools. To get approximate estimate of final complexity we should multiply 1.71 by 2 (it is the same for face-detection-0101). So, face-detection-0103 becomes not optimal in sense of "complexity vs accuracy". But if it good at VPU we still can merge it. We need your opinion.

You don't need to fix complexity numbers. I'll do it by myself for all models later.

@snosov1 @asutic FYI

Reply:
@Ilya-Krylov Generally, face-detection-0103 has a 33% faster performance on VPU than face-detection-0104. Given that our complexity is high, I'll be experimenting with some other model varieties to get a more optimal complexity, while hopefully not sacrificing our accuracy. 

Which OpenVINO tool should I use to get a more accurate complexity for these tests?

Reply:
I'd say the complexity is a somewhat secondary factor to estimate the network performance. We use it as a proxy because we target different CPU/iGPU platforms and it's difficult to measure all of them. In the case of VPU it's more tangible. So, if we can operate with actual inference time - it's, actually, better.

Two things to notice, though:
1. We should, probably, have reasonably high confidence that faster execution despite the GFLOPs difference is _fundamental_ (i.e. due to HW specifics, not poor SW implementation)
2. It makes sense to provide a separate spreadsheet in readme with (Acc/VPU perf) numbers along with (Acc/GFLOPs)

Reply:
@snosov1 

1. What would be the best way to check that faster execution is fundamental?
2. I've been keeping track of VPU performance on MX. Let me know if I should update the readme